examples/b2c/checkout/integrations/classes/B2CDeliverySample.cls

```diff
@@ -95,7 +87,11 @@ global  class B2CDeliverySample implements sfdc_checkout.CartShippingCharges {
                     (String) providerAndRate.get('serviceCode'),
                     (Decimal) providerAndRate.get('shipmentCost'),
                     (Decimal) providerAndRate.get('otherCost'),
-                    (String) providerAndRate.get('serviceName')
+                    (String) providerAndRate.get('serviceName'),
```

Comment:
should this be carrier?

Reply:
Yes .. Our Existing Heroku Service does not have any Carrier field. For this sample, we just populate carrier with the serviceName that we get from Heroku.

Reply:
So the serviceName,carrier,classOfService will all be set as the same

Reply:
The (sample) Heroku Service (as of now) only returns the data:

{
  "status": "calculated",
  "rate": {
    "name": "Delivery Method 1",
    "serviceName": "Test Carrier 1",
    "serviceCode": "SNC9600",
    "shipmentCost": 11.99,
    "otherCost": 5.99
  }
}

Since this is just a sample integration, I just reused the existing data.